doc/source/timeseries.rst

```diff
@@ -402,15 +392,29 @@ using various combinations of parameters like ``start``, ``end``,
 
    pd.bdate_range(start=start, periods=20)
 
-The start and end dates are strictly inclusive. So it will not generate any
-dates outside of those dates if specified.
+.. versionadded:: 0.21.0
+
+``bdate_range`` can also generate a range of custom frequency dates by using
```

Comment:
I think if you indent this paragraph, it more 'belongs' to the versionadded directive (although not fully sure how it will look like when rendered)

Reply:
I tried indenting and it rendered everything on the same line, i.e. "_New in version 0.21.0:_ `bdate_range` can also...".  I changed it back to make it consistent with the other usages of versionadded within `timeseries.rst` and other documentation.